src/ort_genai_c.h

```diff
@@ -273,8 +280,21 @@ OGA_EXPORT void OGA_API_CALL OgaDestroyMultiModalProcessor(OgaMultiModalProcesso
  */
 OGA_EXPORT OgaResult* OGA_API_CALL OgaTokenizerEncode(const OgaTokenizer*, const char* str, OgaSequences* sequences);
 
+/*
+ * \brief Converts the given string to a single token id and adds it to the end of the sequence at the given index.
+          If the sequence does not exist, a new sequence is created at the given index if sequence_idx is equal to the current sequences count.
+ * \param[in] tokenizer The tokenizer to use to convert the string to a token id.
+ * \param[in] str The string to convert to a token id.
+ * \param[in] sequences The OgaSequences to add the token id to.
+ * \param[in] sequence_idx The index of the sequence to add the token id to.
+ * \return OgaResult containing the error message if the conversion of the string to a token id failed.
+ */
+OGA_EXPORT OgaResult* OGA_API_CALL OgaTokenizerToTokenId(const OgaTokenizer* tokenizer, const char* str, OgaSequences* sequences, size_t sequence_idx);
+
```

Comment:
I'm thinking if it is better to return a token id directly instead of add it to OgaSequences. 

We may add an API to allow OgaSequences to append an token id

Reply:
Sure, can do that.

Reply:
Updated the API and example.